### PR TITLE
LPD-32519 Not show warning message after submit return

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
Resend of https://github.com/liferay-commerce/liferay-portal/pull/5226

--- 

https://liferay.atlassian.net/browse/LPD-32519

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:playwright

File: modules/test/playwright/helpers/HeadlessCommerceAdminOrderApiHelper.ts [hunk 1]
```diff
@@ -119,19 +119,29 @@ export class HeadlessCommerceAdminOrderApiHelper {
...
 	async patchOrder(id: number, order: TOrder) {
-		const postOrder = await this.apiHelpers.patch(
+		await this.apiHelpers.patch(
 			`${this.apiHelpers.baseUrl}${this.basePath}orders/${id}?nestedFields=orderItems`,
 			order
 		);
 
+		const patchOrder = await this.apiHelpers.get(
+			`${this.apiHelpers.baseUrl}${this.basePath}orders/${id}`
+		);
```
Comment (smottal) [inline modules/test/playwright/helpers/HeadlessCommerceAdminOrderApiHelper.ts:129]: why do we need to do the get after the patch?

File: modules/test/playwright/tests/commerce/commerce-order-content-web/return.spec.ts [hunk 2]
```diff
@@ -485,3 +485,162 @@ test('LPD-32522 Returns widget displays status field on return items table when
...
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: site.id, type: 'site'});
+
+	const channel = await apiHelpers.headlessCommerceAdminChannel.postChannel({
+		name: getRandomString(),
+		siteGroupId: site.id,
+	});
+
```
Comment (smottal) [inline modules/test/playwright/tests/commerce/commerce-order-content-web/return.spec.ts:507]: probably this empty line is not needed

File: modules/test/playwright/helpers/HeadlessCommerceAdminOrderApiHelper.ts (hunk 1 above)
Comment (andrea-ale-sbarra) [reply modules/test/playwright/helpers/HeadlessCommerceAdminOrderApiHelper.ts:129]: because this patch doesn't return anything

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**